assets/src/scripts/accessibility.js

```diff
@@ -49,7 +49,7 @@ function addSROnlyTable({container, columnNames, data}) {
         .data(function(d) {
             return d;
         })
-        .enter().append('td')
+        .enter().append('th')
```

Comment:
This should not have been changed. It's selecting on td

Reply:
The aXe accessibility validator caught an issue there - that the scope attribute isn't defined on `td`. I suppose the select should be updated also?

Reply:
yes.

Reply:
Maybe it would be more appropriate to remove the scope attribute, since it's not a label for the row.

Reply:
I would be okay with that for now. Jim and I were just talking about how to add the previous year's time series data, whether it should be a separate table or in this table. If we included it in this table, I think it might be appropriate to label the cell containing the date with scope=row. When the second line gets added each <tr> would have two <th> elements one for this year's data and one for last years both with scope=row which is allowed.

Reply:
Ah yes, that makes sense.